app/helpers/application_helper.rb

```diff
@@ -25,7 +25,7 @@ def cover_visible?
   end
 
   def in_repository?
-    params[:repository_id] || params[:controller] == 'repositories'
+    params[:repository_id].present? || params[:controller] == 'repositories'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.